### PR TITLE
Inline svg issue fixed

### LINKED ISSUE #2448: Inline SVG as background image in buttons not aligning correctly
# Issue Description
I noticed in every case an inline svg is used as the background for a button, the svg does not align vertically. Some current implementations work around this by adding margins to control the centering. Some components do not center the svg at all.

# Examples
## [terra-dialog](http://engineering.cerner.com/terra-core/#/site/components/site/dialog/index)
image

## [terra-toggle-button](http://engineering.cerner.com/terra-core/#/site/components/site/toggle-button/index)
image

## terra-action-header (In development)
image

## Issue Type

- [ ] New Feature
- [ ] Enhancement
- [X] Bug
- [ ] Other

## Expected Behavior
Inline SVGs vertically align in terra-button.

## Current Behavior
Inline SVGs are not vertically centered without setting vertical margins.

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Resolves #2448

### CODE REVIEW

**Comment by saket2403** [comment]
Action Header
Dialog
Toggle button

Some IE Screenshots of the changes made.

**Comment by benbcai** [comment]
@saket2403 Are you waiting for UX to review/verify this PR or are you needing input/feedback from UX?

File: packages/terra-action-header/CHANGELOG.md [hunk 1]
```diff
@@ -33,6 +33,9 @@ Unreleased
 
 2.21.0 - (July 17, 2019)
 ------------------
+### Changed
+* Inline SVG aligned for IE and wdio screenshots Updated.
```
Comment (bjankord) [inline packages/terra-action-header/CHANGELOG.md]: Let's update this to:
```suggestion
* Updated `--terra-action-header-icon-top` variable to only apply in Microsoft browsers
```

File: packages/terra-button-group/CHANGELOG.md [hunk 2]
```diff
@@ -42,6 +45,7 @@ Unreleased
 ### Changed
 * Minor dependency version bump
 
+
```
Comment (bjankord) [inline packages/terra-button-group/CHANGELOG.md]: We can remove this extra line

File: packages/terra-button-group/src/terra-dev-site/test/button-group/ButtonGroupIcon.test.jsx [hunk 3]
```diff
@@ -1,11 +1,7 @@
 import React from 'react';
-import classNames from 'classnames/bind';
 import ButtonGroup from '../../../ButtonGroup';
-import styles from './ButtonGroupIcon.test.scss';
```
Comment (bjankord) [inline packages/terra-button-group/src/terra-dev-site/test/button-group/ButtonGroupIcon.test.jsx]: If we no longer need this .scss file, we can delete it.

File: packages/terra-button-group/CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,9 @@ Changelog
 Unreleased
 ----------
 ### Changed
+
+* Inline SVG aligned for IE and wdio screenshots Updated.
+* Import for package version from package.json added to doc page
```
Comment (bjankord) [inline packages/terra-button-group/CHANGELOG.md]: This line doesn't seem to relate to the changes in the PR.

File: packages/terra-button/CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,8 @@ Changelog
 Unreleased
 ----------
 ### Changed
+* Inline SVG aligned for IE and wdio screenshots Updated.
+* Import for package version from package.json added to doc page
```
Comment (bjankord) [inline packages/terra-button/CHANGELOG.md]: This line seem unrelated to the changes in this package.

File: packages/terra-button/src/Button.module.scss [hunk 6]
```diff
@@ -24,16 +24,19 @@
...
     cursor: pointer;
-    display: inline-block;
+    display: inline-flex;
     font-family: inherit;
     font-size: var(--terra-button-font-size, 1rem);
+    justify-content: center;
     line-height: var(--terra-button-line-height, 1.429);
     margin: 0; /* Reset margin for Safari. Safari defaults to 2px. */
     max-width: 100%; /* Need to help with overflow-wrap */
+    min-height: 2rem;
```
Comment (bjankord) [inline packages/terra-button/src/Button.module.scss]: I noticed some of the icon buttons went from being 30px tall to 28px tall. I think we should set this min-height to `2.1428rem` to help ensure the buttons stay 30px tall, the same height as non-icon buttons.

File: packages/terra-dialog/CHANGELOG.md [hunk 7]
```diff
@@ -50,6 +50,9 @@ Unreleased
 
 2.15.0 - (June 28, 2019)
 ------------------
+### Changed
+* Inline SVG aligned for IE.
```
Comment (bjankord) [inline packages/terra-dialog/CHANGELOG.md]: Let's update this to:
```suggestion
* Updated `--terra-dialog-header-close-icon-margin-top` variable to only apply in Microsoft browsers
```

File: packages/terra-action-header/src/ActionHeader.module.scss [hunk 8]
```diff
@@ -14,7 +14,10 @@
     background-size: contain;
     display: inline-block;
     position: relative;
-    top: var(--terra-action-header-icon-top, 0.1429rem);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      top: var(--terra-action-header-icon-top, 0.1429rem);
```
Comment (bjankord) [inline packages/terra-action-header/src/ActionHeader.module.scss]: Let's rename this variable to the following to denote its scoped to Microsoft vendor prefix
```suggestion
      top: var(--terra-action-header-icon-ms-top, 0.1429rem);
```

File: packages/terra-dialog/src/Dialog.module.scss [hunk 9]
```diff
@@ -32,9 +32,11 @@
     background: var(--terra-dialog-header-close-background, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='rgba(17, 17, 17, 1)' d='M28.2 24L42.9 9.1 40.8 7l-1.7-1.6-.4-.5L24 19.7 9.4 4.9 7.2 7 5.6 8.6l-.5.5L19.8 24 5.1 38.9 7.2 41l1.7 1.6.5.5L24 28.3l14.7 14.8.4-.5 1.7-1.6 2.1-2.1L28.2 24z'/></svg>"));
     display: inline-block;
     height: var(--terra-dialog-header-close-icon-height, 1.14286rem);
-    margin-top: var(--terra-dialog-header-close-icon-margin-top, -0.214285rem);
     vertical-align: middle;
     width: var(--terra-dialog-header-close-icon-width, 1.14286rem);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      margin-top: var(--terra-dialog-header-close-icon-margin-top, 0.07rem);
```
Comment (bjankord) [inline packages/terra-dialog/src/Dialog.module.scss]: Let's rename this variable to the following to denote its scoped to Microsoft vendor prefix
```suggestion
      margin-top: var(--terra-dialog-header-close-icon-margin-ms-top, 0.07rem);
```

File: packages/terra-action-header/src/ActionHeader.module.scss (hunk 8 above)
Comment (benbcai) [reply packages/terra-action-header/src/ActionHeader.module.scss]: But renaming it would be a breaking change.

Comment (benbcai) [reply packages/terra-action-header/src/ActionHeader.module.scss]: Talked to Brett offline...if we proactively update this variable in the other themes, then it wouldn't break them and should be ok.

**Comment by saket2403** [comment]
> @saket2403 Are you waiting for UX to review/verify this PR or are you needing input/feedback from UX?

@benbcai I want UX to review the PR, if the changes look good enough for IE and provide any input otherwise.

File: packages/terra-button-group/src/terra-dev-site/test/button-group/ButtonGroupIcon.test.jsx (hunk 3 above)
Comment (saket2403) [reply packages/terra-button-group/src/terra-dev-site/test/button-group/ButtonGroupIcon.test.jsx]: Deleted in 2ff8a50

File: packages/terra-action-header/CHANGELOG.md (hunk 1 above)
Comment (saket2403) [reply packages/terra-action-header/CHANGELOG.md]: Changed in 2ff8a50

File: packages/terra-button-group/CHANGELOG.md (hunk 2 above)
Comment (saket2403) [reply packages/terra-button-group/CHANGELOG.md]: Resolved in 2ff8a50

File: packages/terra-button-group/src/terra-dev-site/test/button-group/ButtonGroupIcon.test.jsx (hunk 3 above)
Comment (saket2403) [reply packages/terra-button-group/src/terra-dev-site/test/button-group/ButtonGroupIcon.test.jsx]: Resolved in 2ff8a50

File: packages/terra-button-group/CHANGELOG.md (hunk 4 above)
Comment (saket2403) [reply packages/terra-button-group/CHANGELOG.md]: Resolved in 2ff8a50

File: packages/terra-button/src/Button.module.scss (hunk 6 above)
Comment (saket2403) [reply packages/terra-button/src/Button.module.scss]: Resolved in 2ff8a50

File: packages/terra-dialog/CHANGELOG.md (hunk 7 above)
Comment (saket2403) [reply packages/terra-dialog/CHANGELOG.md]: Resolved in 2ff8a50

File: packages/terra-action-header/src/ActionHeader.module.scss (hunk 8 above)
Comment (saket2403) [reply packages/terra-action-header/src/ActionHeader.module.scss]: Resolved in 2ff8a50

File: packages/terra-dialog/src/Dialog.module.scss (hunk 9 above)
Comment (saket2403) [reply packages/terra-dialog/src/Dialog.module.scss]: Resolved in 2ff8a50

File: packages/terra-button/CHANGELOG.md (hunk 5 above)
Comment (saket2403) [reply packages/terra-button/CHANGELOG.md]: Resolved in 2ff8a50

File: packages/terra-action-header/CHANGELOG.md [hunk 10]
```diff
@@ -3,10 +3,12 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Updated `--terra-action-header-icon-top` variable to only apply in Microsoft browsers.
```
Comment (bjankord) [inline packages/terra-action-header/CHANGELOG.md]: Let's update this to say something along the lines of "Removed `--terra-action-header-icon-top` variable, added `--terra-action-header-icon-ms-top` variable to only apply top styling in Microsoft browsers.

File: packages/terra-dialog/CHANGELOG.md [hunk 11]
```diff
@@ -3,10 +3,12 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Updated `--terra-dialog-header-close-icon-margin-top` variable to only apply in Microsoft browsers
```
Comment (bjankord) [inline packages/terra-dialog/CHANGELOG.md]: Let's update this to say something along the lines of "Removed `--terra-dialog-header-close-icon-margin-top` variable, added `--terra-dialog-header-close-icon-ms-margin-top` variable to only apply top styling in Microsoft browsers.

File: packages/terra-action-header/CHANGELOG.md (hunk 10 above)
Comment (saket2403) [reply packages/terra-action-header/CHANGELOG.md]: Updated in d761f5e

File: packages/terra-dialog/CHANGELOG.md (hunk 11 above)
Comment (saket2403) [reply packages/terra-dialog/CHANGELOG.md]: Updated in d761f5e

File: packages/terra-action-header/src/ActionHeader.module.scss [hunk 12]
```diff
@@ -14,7 +14,10 @@
     background-size: contain;
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-action-header/src/ActionHeader.module.scss]: Resolved in 3cfa856

File: packages/terra-action-header/src/ActionHeader.module.scss [hunk 13]
```diff
@@ -14,7 +14,10 @@
...
     display: inline-block;
     position: relative;
-    top: var(--terra-action-header-icon-top, 0.1429rem);
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-action-header/src/ActionHeader.module.scss]: Resolved in 3cfa856

File: packages/terra-action-header/src/ActionHeader.module.scss [hunk 14]
```diff
@@ -14,7 +14,10 @@
...
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      top: var(--terra-action-header-icon-ms-top, 0.1429rem);
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-action-header/src/ActionHeader.module.scss]: Resolved in 3cfa856

**Comment by benbcai** [comment]
Is this pretty much ready for final UX review?

File: packages/terra-action-header/CHANGELOG.md [hunk 15]
```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* Updated `--terra-action-header-icon-top` variable, added `--terra-action-header-icon-ms-top` variable to only apply top styling in Microsoft browsers.
```
Comment (avinashg1994) [inline packages/terra-action-header/CHANGELOG.md]: This should be updated to "**Removed** `--terra-dialog-header-close-icon-margin-top` ..." as per https://github.com/cerner/terra-core/pull/2523#discussion_r321389206

Comment (saket2403) [reply packages/terra-action-header/CHANGELOG.md]: `--terra-action-header-icon-top` variable was not removed just value was updated as per this [comment](https://github.com/cerner/terra-core/pull/2523/files#r325135405)

File: packages/terra-button/CHANGELOG.md [hunk 16]
```diff
@@ -16,7 +18,6 @@ Unreleased
 
 3.22.0 - (September 3, 2019)
 ------------------
-### Changed
```
Comment (avinashg1994) [inline packages/terra-button/CHANGELOG.md]: I think this was accidentally removed, we should add it back.

**Comment by avinashg1994** [comment]
There are alignment issues on IE11 browser:
1. **Action Header**:
Screenshot 2019-10-03 at 2 02 00 PM
2. **Button Group**:
Button Group
3. **Button**: Even though this is an improvement from before, but a little more tinkering is required
Button
4. **Dialog**: The bottom padding is more than the top padding
Dialog
5. **Toggle Button**: The top padding is more than the bottom padding
Toggle Button

All the above alignment issues are only with the IE browser and works fine in Edge and Chrome.

File: packages/terra-button/CHANGELOG.md (hunk 16 above)
Comment (saket2403) [reply packages/terra-button/CHANGELOG.md]: Fixed in 7593ab7

**Comment by saket2403** [comment]
> All the above alignment issues are only with the IE browser and works fine in Edge and Chrome.

Resolved in 27f5e88 and c3cd8d8

File: packages/terra-dialog/src/Dialog.module.scss [hunk 17]
```diff
@@ -32,9 +32,11 @@
     background: var(--terra-dialog-header-close-background, inline-svg("<svg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 48 48'><path fill='rgba(17, 17, 17, 1)' d='M28.2 24L42.9 9.1 40.8 7l-1.7-1.6-.4-.5L24 19.7 9.4 4.9 7.2 7 5.6 8.6l-.5.5L19.8 24 5.1 38.9 7.2 41l1.7 1.6.5.5L24 28.3l14.7 14.8.4-.5 1.7-1.6 2.1-2.1L28.2 24z'/></svg>"));
     display: inline-block;
     height: var(--terra-dialog-header-close-icon-height, 1.14286rem);
-    margin-top: var(--terra-dialog-header-close-icon-margin-top, -0.214285rem);
     vertical-align: middle;
     width: var(--terra-dialog-header-close-icon-width, 1.14286rem);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      margin-top: var(--terra-dialog-header-close-icon-ms-margin-top, 0.07rem);
+    }
```
Comment (neilpfeiffer) [inline packages/terra-dialog/src/Dialog.module.scss]: New sizing approach to match action-header, new adjustments for everything after `background: var(--...` :
```
    background-repeat: no-repeat;
    background-size: auto;
    display: inline-block;    
    height: var(--terra-dialog-header-close-icon-height, 1rem);
    margin-top: var(--terra-dialog-header-close-icon-margin-top, 0);
    vertical-align: var(--terra-dialog-header-close-icon-vertical-align, -0.15rem);
    width: var(--terra-dialog-header-close-icon-width, 1rem);
    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
      vertical-align: var(--terra-dialog-header-close-icon-vertical-align, -0.15rem);
    }
```

File: packages/terra-toggle-button/src/ToggleButton.module.scss [hunk 18]
```diff
@@ -5,10 +5,19 @@
 
   .icon {
     display: inline-block;
-    font-size: 0.8em;
+    line-height: 1rem;
```
Comment (neilpfeiffer) [inline packages/terra-toggle-button/src/ToggleButton.module.scss]: no unit of measure
```suggestion
    line-height: 1;
```

File: packages/terra-toggle-button/src/ToggleButton.module.scss [hunk 19]
```diff
@@ -5,10 +5,19 @@
...
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      line-height: 1.35rem;
+    }
 
+    svg {
+      font-size: 0.8em;
+      position: relative;
+      top: -0.15em;
+    }
+  }
```
Comment (neilpfeiffer) [inline packages/terra-toggle-button/src/ToggleButton.module.scss]: Media-query applied in the wrong spot, updated values:
```
    svg {
      font-size: 0.8em;
      position: relative;
      top: -0.07143em;
      @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
          top: -0.14285em;
      }
    }
  }
```

File: packages/terra-button/src/Button.module.scss [hunk 20]
```diff
@@ -140,10 +145,14 @@
...
     display: inline-block;
+    line-height: 0;
   }
 
   .icon-svg {
     height: var(--terra-button-icon-height, 1rem);
     width: var(--terra-button-icon-width, 1rem);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      margin-top: var(--terra-button-icon-ms-margin-top, 0.1429em);
+    }
```
Comment (neilpfeiffer) [inline packages/terra-button/src/Button.module.scss]: can remove - adjustments applied elsewhere now so not needed here.

File: packages/terra-button/src/Button.module.scss [hunk 21]
```diff
@@ -24,16 +24,19 @@
   // stylelint-enable selector-no-qualifying-type
 
   .button {
+    align-items: center;
     border-radius: var(--terra-button-border-radius, 0.2142rem);
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-button/src/Button.module.scss]: Resolved in 5ebdfe6

File: packages/terra-button/src/Button.module.scss [hunk 22]
```diff
@@ -104,11 +107,12 @@
 
   // Button style modifiers
   .block {
-    display: block;
+    display: flex;
     width: 100%;
   }
 
   .compact {
+    min-height: 1.429rem;
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-button/src/Button.module.scss]: Resolved in 5ebdfe6

File: packages/terra-action-header/src/ActionHeader.module.scss [hunk 23]
```diff
@@ -11,10 +11,14 @@
...
     background-repeat: no-repeat;
-    background-size: contain;
+    background-size: auto;
     display: inline-block;
     position: relative;
-    top: var(--terra-action-header-icon-top, 0.1429rem);
+    top: var(--terra-action-header-icon-top, 0);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      margin-top: var(--terra-action-header-icon-ms-margin-top, 0.21428rem);
+    }
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-action-header/src/ActionHeader.module.scss]: Resolved in 5ebdfe6

File: packages/terra-button/src/Button.module.scss [hunk 24]
```diff
@@ -120,6 +124,7 @@
   }
 
   .icon-only {
+    line-height: 0;
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-button/src/Button.module.scss]: Resolved in 5ebdfe6

File: packages/terra-button/src/Button.module.scss [hunk 25]
```diff
@@ -140,10 +145,14 @@
 
   .icon {
     display: inline-block;
+    line-height: 0;
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-button/src/Button.module.scss]: Resolved in 5ebdfe6

File: packages/terra-dialog/src/Dialog.module.scss (hunk 17 above)
Comment (saket2403) [reply packages/terra-dialog/src/Dialog.module.scss]: Resolved in 5ebdfe6

File: packages/terra-toggle-button/src/ToggleButton.module.scss (hunk 18 above)
Comment (saket2403) [reply packages/terra-toggle-button/src/ToggleButton.module.scss]: Resolved in 5ebdfe6

File: packages/terra-toggle-button/src/ToggleButton.module.scss (hunk 19 above)
Comment (saket2403) [reply packages/terra-toggle-button/src/ToggleButton.module.scss]: Resolved in 5ebdfe6

File: packages/terra-button/src/Button.module.scss (hunk 20 above)
Comment (saket2403) [reply packages/terra-button/src/Button.module.scss]: Resolved in 5ebdfe6

File: packages/terra-button/src/Button.module.scss [hunk 26]
```diff
@@ -24,16 +24,19 @@
...
     cursor: pointer;
-    display: inline-block;
+    display: inline-flex;
     font-family: inherit;
     font-size: var(--terra-button-font-size, 1rem);
+    justify-content: center;
     line-height: var(--terra-button-line-height, 1.429);
     margin: 0; /* Reset margin for Safari. Safari defaults to 2px. */
     max-width: 100%; /* Need to help with overflow-wrap */
+    min-height: 2.1428rem;
```
[the remark this answers is not in the document]
Comment (saket2403) [reply packages/terra-button/src/Button.module.scss]: Resolved in 5ebdfe6

File: packages/terra-action-header/src/ActionHeader.module.scss [hunk 27]
```diff
@@ -11,10 +11,14 @@
...
     background-repeat: no-repeat;
-    background-size: contain;
+    background-size: auto;
     display: inline-block;
     position: relative;
-    top: var(--terra-action-header-icon-top, 0.1429rem);
+    top: var(--terra-action-header-icon-top, 0);
+    vertical-align: var(--terra-action-header-icon-vertical-align, -0.14285rem);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      vertical-align: var(--terra-action-header-icon-ms-vertical-align, -0.21428rem);
```
Comment (neilpfeiffer) [inline packages/terra-action-header/src/ActionHeader.module.scss]:
```suggestion
      vertical-align: var(--terra-action-header-icon-ms-vertical-align, -0.19rem);
```

File: packages/terra-toggle-button/src/ToggleButton.module.scss [hunk 28]
```diff
@@ -5,10 +5,19 @@
...
   .icon {
     display: inline-block;
-    font-size: 0.8em;
+    line-height: 1;
     transform: rotate(0deg);
+    
+    svg {
+      font-size: 0.8em;
+      position: relative;
+      top: -0.07143em;
```
Comment (neilpfeiffer) [inline packages/terra-toggle-button/src/ToggleButton.module.scss]:
```suggestion
      top: -0.14em;
```

File: packages/terra-toggle-button/src/ToggleButton.module.scss [hunk 29]
```diff
@@ -5,10 +5,19 @@
...
+      @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+        top: -0.14285em;
```
Comment (neilpfeiffer) [inline packages/terra-toggle-button/src/ToggleButton.module.scss]:
```suggestion
        top: -0.07em;
```

File: packages/terra-action-header/src/ActionHeader.module.scss (hunk 27 above)
Comment (saket2403) [reply packages/terra-action-header/src/ActionHeader.module.scss]: Resolved in 898cfef

File: packages/terra-toggle-button/src/ToggleButton.module.scss (hunk 28 above)
Comment (saket2403) [reply packages/terra-toggle-button/src/ToggleButton.module.scss]: Resolved in 898cfef

File: packages/terra-toggle-button/src/ToggleButton.module.scss (hunk 29 above)
Comment (saket2403) [reply packages/terra-toggle-button/src/ToggleButton.module.scss]: Resolved in 898cfef

**Comment by neilpfeiffer** [comment]
Tested: PR #2523
Resolves: Issue #2448
 
**Reviewed:**
- Verified updates to Button, Button-Group, Action-Header, Dialog, Toggle-Button in IE as having better vertical alignment than current.
   
\
Verified by @neilpfeiffer, ready for Release.

**Pass** |
-----------|
UX Review|

\
\
@saket2403  @ryanthemanuel : Ready to Merge

File: packages/terra-dialog/src/Dialog.module.scss [hunk 30]
```diff
@@ -30,11 +30,16 @@
...
-    height: var(--terra-dialog-header-close-icon-height, 1.14286rem);
-    margin-top: var(--terra-dialog-header-close-icon-margin-top, -0.214285rem);
-    vertical-align: middle;
-    width: var(--terra-dialog-header-close-icon-width, 1.14286rem);
+    height: var(--terra-dialog-header-close-icon-height, 1rem);
+    margin-top: var(--terra-dialog-header-close-icon-margin-top, 0);
+    vertical-align: var(--terra-dialog-header-close-icon-vertical-align, -0.15rem);
+    width: var(--terra-dialog-header-close-icon-width, 1rem);
+    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
+      vertical-align: var(--terra-dialog-header-close-icon-vertical-align, -0.15rem);
```
Comment (avinashg1994) [inline packages/terra-dialog/src/Dialog.module.scss]: @neilpfeiffer @saket2403 ,
Do we need to differentiate this theme variable from the normal vertical-align theme variable ([here](https://github.com/cerner/terra-core/pull/2523/files?file-filters%5B%5D=.scss#diff-29f4c6651a36c677ff2dcc4d9a2770d6R38)). 

Something similar to as seen [here](https://github.com/cerner/terra-core/pull/2523/files?file-filters%5B%5D=.scss#diff-b2ae2172c829371a523555d2e0626b06R18-R21)
